source/releasenotes/compat.rst

```diff
@@ -23,55 +23,51 @@ This section lists the operating systems that are supported for running
 CloudStack Management Server.
 
 -  Ubuntu 20.04 LTS, 22.04 LTS, 24.04 LTS
--  CentOS versions 7
--  Rocky Linux 8, 9
--  Alma Linux 8, 9
 -  Oracle Linux 8, 9
--  RHEL versions 7, 8, 9
+-  Alma Linux 8, 9
+-  Rocky Linux 8, 9
+-  RHEL versions 8, 9
 -  openSUSE Leap 15
 -  SUSE Linux Enterprise Server 15 (not tested, but expected to work same as with openSUSE 15)
+-  Debian 12 (not tested, but expected to work same as Ubuntu)
 
-   .. note:: There is a known issue with ipmitool with RHEL8 / SUSE, so certain functionality such as out of band management might not work
+   .. note:: There is a known issue with ipmitool with EL8 / EL9 / SUSE, so certain functionality such as out of band management might not work
 
 Software Requirements
 ~~~~~~~~~~~~~~~~~~~~~
 
--  Java JRE 17
--  MySQL 5.6, 5.7, 8.0
+-  Java JRE 11 or Java JRE 17
```

Comment:
```suggestion
-  Java JRE 17
```

for main/4.20, the java/jre version must be 17 if OS is centos8/9 or ubuntu

ubuntu
https://github.com/apache/cloudstack/blob/b69cc0272d48f0aea7353627d760c27c284dad84/debian/control#L20
https://github.com/apache/cloudstack/blob/b69cc0272d48f0aea7353627d760c27c284dad84/debian/control#L27

centos8/9:
https://github.com/apache/cloudstack/blob/b69cc0272d48f0aea7353627d760c27c284dad84/packaging/centos8/cloud.spec#L55
https://github.com/apache/cloudstack/blob/b69cc0272d48f0aea7353627d760c27c284dad84/packaging/centos8/cloud.spec#L101

Reply:
As this is merged now, feel free to raise a new PR to correct this